extra-dev/packages/rocq-hierarchy-builder/rocq-hierarchy-builder.dev/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi" ]
+license: "MIT"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
+
+build: [ [ make "build"]
+         [ make "test-suite" ] {with-test & rocq-core:installed}
```

Comment:
rocq-core is in the depends, what's the point of checking if it's installed here?

Reply:
I misread rocq as coq somehow